src/components/pages/Home.js

```diff
@@ -48,9 +48,9 @@ class Home extends React.Component {
                     <div className="speakers-home-link mr-auto align-self-center">
                         <h3><Link to='/speakers' className='nav-link'><span className="header-gradient">Speakers</span></Link></h3>
```

Comment:
This text is repetitive, though it does link to the speaker page but we can remove this and add the link to speakers page fto the h3 tag for speakers section below.